modules/optimizing-mtu-networking.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-networking.adoc
+
+[id="optimizing-mtu_{context}"]
+= Optimizing the MTU for your network
+
+There are two important maximum transmission units (MTUs): the network interface card (NIC) MTU and the SDN overlay's MTU.
```

Comment:
We seem to have excised _SDN_ mostly from the docs, because reasons. _OpenShift SDN_ is a legacy name. Lately, it's called the cluster network provider. (Such as _OpenShift SDN cluster network provider_.) So I'd say

> and the cluster network MTU.